Cargo.toml

```diff
@@ -24,6 +24,8 @@ members = [
 
 [dependencies]
 appendlist = "1.4"
+# Intentionally pick one commit back from the "release 0.37.1" branch since the release commit is likely to be rebased
+ash = { git = "https://github.com/ash-rs/ash", rev = "28a4868dc302013e00569dc5393f8a0b43ed4a70", optional = true }
```

Comment:
Why not pick a commit from the `0.37-stable` branch, which intentionally exists as a sort of "master" branch but for "backporting" non-breaking changes?

`0.37.1-release~` is also part of that branch, all you need to do is refine the comment.